fiftyone/utils/torch.py

```
@@ -5,15 +5,14 @@
-import logging
+import functools
 import itertools
+import logging
 import multiprocessing
 import os
-import sys
-from typing import Callable, Any, Optional, List
 import pickle
-import functools
+import sys
+from typing import Any, Optional, List
```

Flake8 (7.2.0)

[error] 'typing.Any' imported but unused

(F401)